prometheus-openstack-exporter

```diff
@@ -618,7 +640,7 @@ class OpenstackExporterHandler(BaseHTTPRequestHandler):
                 self.send_header('Content-Type', CONTENT_TYPE_LATEST)
                 self.end_headers()
                 self.wfile.write(output)
-            except:
+            except:  # noqa: E722
```

Comment:
I kept this because swift's validate_configuration() will exit if it fails.  I'll deal with this in the Swift account-usage branch by some other means.